src/content/core/okteto-variables.mdx

```diff
@@ -7,36 +7,259 @@ id: okteto-variables
+and share config values using Okteto Variables.
 
-Okteto variables help you save application configuration in Okteto and automatically inject it during deployment time.
+This page is a guide for those who are authoring Okteto manifests and configuring Okteto for their team and organization. If you are interested in runtime variables of dev environments, see [Development Containers](/docs/development/containers).
 
-There are a few different ways to leverage variables. In short:
+## What are Okteto Variables?
 
-- You can define variables manually in the manifest or in the Okteto UI
-- There are some default variables built in to the Okteto platform
-- All variables used in the Okteto manifest are automatically expanded
-- All variables are injected as runtime environment variables in the execution of your `deploy` and `destroy` scripts
+Okteto Variables are a way to parametrize your [Okteto manifest](/docs/reference/okteto-manifest) and the deploy, destroy and test strategy.
 
-We will cover each of these behaviors in more detail below.
+### Types of Variables
 
-## User-defined variables
+Each type is designed to solve a specific use-case.
 
-Okteto supports manual definition of variables, offering significant flexibility in configuration. These user-defined variables are separate from Okteto's default, built-in variables. We recommend creating variables in the following order of preference:
+- **Managed by Okteto**: variables injected in your dev environment automatically by Okteto
+- **The `--var` flag**: Used to define variables ad-hoc when using the Okteto CLI
+- **Catalog**: Used to define project-specific variables
+- **Local Environment Variables**: Used to define variables ad-hoc when using the Okteto CLI
+- **The `.env` file**: Used to define project or team-specific variables
+- **User Variables**: Used to define variables at the user level
+- **Admin Variables**: Used to define variables for all users in the organization
 
-- Local variables
-- Deploy time variables
-- Catalog variables
-- User variables (in settings)
-- [Admin variables](admin/dashboard.mdx#admin-variables)
+:::note
+
+The above listing order is also the order in which the Okteto CLI gives precedence to each variable type. At the top the variables with the highest precedence.
+
+Variables with the same name defined in different types will be overridden by the variable with higher precedence.
+
+Example: the var `PORT` defined both in `.env` as `PORT=3000` and as `okteto deploy --var PORT=4000`, the value of `PORT` will be `4000` because the `--var` flag has higher precedence.
+
+:::
+
+
+### Scope
+
+Okteto Variables are scoped to the execution of the Okteto CLI. This means that they are:
+
+- available for expanding values in the Okteto manifest
+- available in the `deploy`, `destroy` and `test` commands in the Okteto manifest
```

Comment:
I think this one is redundant

Reply:
do you mean because the first item already covers it?

I used a slighly redundant writing to make sure to cover all cases and remove ambiguity.

Let me know how you would change it. Remove or incorporate into a single item? @pchico83

Reply:
I think the second one is redundant with the third one

Reply:
you're right, too redundant. Made this change:

```
-- available in the `deploy`, `destroy` and `test` commands in the Okteto manifest
-- available to scripts and tools running as part of `deploy`, `destroy` and `test` commands in the Okteto manifest
+- available to commands, scripts and tools running as part of `deploy`, `destroy` and `test` commands in the Okteto manifest
```

Perhaps it's obvious, I hope it's clear that Okteto Variables are available both in commands but also internet tooling invoked by such commands.